pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -137,7 +138,15 @@ func Execute(configFile io.Reader) {
 	app.Config.HTTP.Headers.Set("X-Registry-Supports-Signatures", "1")
 
 	app.RegisterHealthChecks()
-	handler := alive("/", app)
+	handler := http.Handler(app)
```

Comment:
It should, but our read registry traffic is an order of magnitude than our write load.  Setting a read rate limit is going to dwarf the write limit, which mean we'll have to set the read limit too low to preserve our current scale.

If pull though proxy needs to be limited, then it's possible we need another limiter wired in (or connect this limiter to that).  However, if this *only* addresses non-pullthroigh it gets us closer to not broken.  We aren't breaking today because of pullthrough though.